src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -1192,6 +1192,9 @@ private static int GetOperatorPrecedence(TokenType tokenType)
                 case TokenType.LogicalOr:
                     return 40;
 
+                case TokenType.DoubleQuestion:

```

Comment:
Please could you also add this to the language grammar defined in `grammar.md`?

Reply:
Can you also update Parser unit tests to assert on associativity and precedence?

---
In reply to: [572962340](https://github.com/Azure/bicep/pull/1489#discussion_r572962340) [](ancestors = 572962340)

Reply:
And can you update `docs/spec/expressions.md` to add the new operator?

---
In reply to: [573181930](https://github.com/Azure/bicep/pull/1489#discussion_r573181930) [](ancestors = 573181930,572962340)

Reply:
@anthony-c-martin to be honest, I don't know what to write it in grammar.md 😢 

Would this be ok?
```
coalesce -> expression ?? coalesce | expression
```

Reply:
Actually, it looks like it's somehow already been defined in `docs/grammar.md`! So you can just update `docs/spec/expressions.md`. Sorry for the confusion.

![image](https://user-images.githubusercontent.com/38542602/107428605-e5800600-6af0-11eb-8243-3b34568e880e.png)

Reply:
@anthony-c-martin @majastrz  - docs and tests updated.